content/docs/2.7/concepts/scaling-deployments.md

```diff
@@ -195,6 +192,21 @@ Starting from Kubernetes v1.18 the autoscaling API allows scaling behavior to be
 
 **Assumptions:** KEDA must be running on Kubernetes cluster v1.18+, in order to be able to benefit from this setting.
 
+---
+#### triggers
+```yaml
+  triggers:
+  # {list of triggers to activate scaling of the target resource}
+```
+
+Trigger fields:
+- **type**: Mandatory. The trigger type.
+- **metadata**: Mandatory. A mapping of key-value parameters that describe the trigger.
+- **authenticationRef**: Optional. A reference to the `TriggerAuthentication` or `ClusterTriggerAuthentication` object that is used to authenticate the scaler with the environment. More details can be found [here](/concepts/authentication).
+- **metricType**: Optional. For all scalers other than CPU and memory, supported metric types are `AverageValue` and `Value`, with `AverageValue` as a default value. For CPU and memory scalers, `AverageValue` and `Utilization` are supported. Refer to the [HPA docs](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale-walkthrough/) for how HPA calculates `replicaCount` based on metric type and value.
```

Comment:
It would be great if we can add examples on metricType, ie. We have this number of replicas, we receive XYZ metric from the trigger and  metriType is set to `Value` then the target number of replicas will be approx XYZ.
And the same for `AverageValue`. So users undertand the difference.